build.gradle

```diff
@@ -108,8 +108,8 @@ subprojects {
 
     check.dependsOn localeTest
 
-    sourceCompatibility = JavaVersion.VERSION_1_8
-    targetCompatibility = JavaVersion.VERSION_1_8
+    sourceCompatibility = JavaVersion.VERSION_11
+    targetCompatibility = JavaVersion.VERSION_11
```

Comment:
Our source code and minimum supported version is `1.8`.

Reply:
Aware.